packages/server-wallet/src/wallet/store.ts

```diff
@@ -106,22 +107,22 @@ export const Store = {
   },
 };
 
-class StoreError extends Error {
-  readonly type = 'InvariantError';
-  constructor(reason: StoreErrors, public readonly data: any = undefined) {
+class StoreError extends WalletError {
+  readonly type = WalletError.errors.InvariantError;
```

Comment:
A bit off topic, but `InvariantError` doesn't quite seem right. Is "not being a participant in the channel" an invariant error?

Reply:
Renaming to `StoreError` for now: 2c7e4b77a